che-editors.yaml

```diff
@@ -562,6 +562,8 @@ editors:
           volumeMounts:
             - name: projector-volume
               path: /projector
+            - name: projector-configuration
+              path: /home/user/.jetbrains
```

Comment:
how does it work on a custom image ?
b/c home folder may be different than /home/user ?

Reply:
With the custom image, when user folder is different than /home/user, it can be any folder, but it should be the same as it provided by `PROJECTOR_CONFIG_DIR` env variable. Although it can be any directory with write permissions.